Solutions/Phosphorus/Data Connectors/PhosphorusDataConnector.json

```diff
@@ -0,0 +1,144 @@
+                    "id":"Phosphorus_Polling",
+                    "publisher": "Phosphorus Inc.",
+                    "descriptionMarkdown": "The Phosphorus Device Connector provides the capability to Phosphorus to ingest device data logs into Microsoft Sentinel through the Phosphorus REST API. The Connector provides visibility into the devices enrolled in Phosphorus. This Data Connector pulls devices information along with its corresponding alerts.",
+                    "graphQueriesTableName": "Phosphorus_CL",
+                    "graphQueries": [
+                        {
+                            "metricName": "Total events received",
+                            "legend": "Phosphorus Audit events",
+                            "baseQuery": "Phosphorus_CL"
+                        }
+                    ],
+                    "dataTypes": [
+                        {
+                            "name": "Phosphorus_CL",
+                            "lastDataReceivedQuery": "Phosphorus_CL \n | summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+                        }
+                    ],
+                    "connectivityCriteria": [
+                        {
+                            "type": "SentinelKindsV2",
+                            "value": [
+                                "APIPolling"
+                            ]
+                        }
+                    ],
+                    "availability": {
+                        "status": 1,
+                        "isPreview": false
+                    },
+                    "permissions": {
+                        "resourceProvider": [
+                            {
+                                "provider": "Microsoft.OperationalInsights/workspaces",
+                                "permissionsDisplayText": "read and write permissions are required.",
+                                "providerDisplayName": "Workspace",
+                                "scope": "Workspace",
+                                "requiredPermissions": {
+                                    "action": true,
+                                    "write": true,
+                                    "read": true,
+                                    "delete": true
+                                }
+                            }
+                        ],
+                        "customs": [
+                            {
+                                "name": "REST API Credentials/permissions",
+                                "description": "**Phosphorus API Key** is required. Please make sure that the API Key associated with the User has the Manage Settings permissions enabled.\n\n Follow these instructions to enable Manage Settings permissions.\n 1. Log in to the Phosphorus Application\n 2. Go to 'Settings' -> 'Groups'\n 3. Select the Group the Integration user is a part of\n 4. Navigate to 'Product Actions' -> toggle on the 'Manage Settings' permission. "
```

Comment:
Can we add a hyperlink here
So that user can  directly visit the domain by clicking on the hyperlink mentioned in the Data connector.

Reply:
The Phosphorus Application's Domain Name is unique for each customer. So its not possible to know beforehand the customer's domain name to add the hyperlink. But all of our customer are aware of the Phosphorus API Key and will be able to perform the required steps without further direction.